internal/signaling/peer.go

```diff
@@ -403,6 +397,7 @@ func (p *Peer) HandleJoinPacket(ctx context.Context, packet JoinPacket) error {
 	}
 
 	p.Lobby = packet.Lobby
+	p.store.Subscribe(ctx, p.Game+p.Lobby, p.ForwardMessage)
```

Comment:
I think it smart to move the `lobby, err := p.store.GetLobby(ctx, p.Game, packet.Lobby)` below the store.Subscribe() calls. That way we'll never miss a peer when to peers connect at the same time.

Reply:
Good one.